blkn/src/Typography/index.js

```diff
@@ -19,6 +19,7 @@ type Props = {
   size: "large" | "small" | "normal",
   type: "primary" | "secondary" | "attention" | "error",
   children: React.Node,
+  color?: string,
```

Comment:
is it desired to introduce colour? I thought all possible states should be achievable by combination of `type` and `variant` https://zpl.io/VxBkGZW

Reply:
I just created PR to fix colors 🙂

Reply:
How do you make use of `variant` with the `Typography` component?
I think it's good to have a handy set of colors `primary`, `secondary`, etc and maximize its use. But imho having an optional `color` prop makes the `Typography` component much more re-usable at a very low cost. (*End of salespitch*) :smile:

Reply:
We'll be bundling Theme support in the future. #35